gothic/gothic.go

```diff
@@ -159,7 +159,18 @@ var CompleteUserAuth = func(res http.ResponseWriter, req *http.Request) (goth.Us
 		return goth.User{}, err
 	}
 
-	return provider.FetchUser(sess)
+	user, err := provider.FetchUser(sess)
+	//Added to test if refresh token implementation is working properly or not
```

Comment:
This probably shouldn't be in `gothic`. `gothic` is meant to be used in applications, not a place to test things. If this was here people would always be refreshing tokens in production when they don't mean to be.